curated_transformers/models/embeddings.py

```diff
@@ -57,18 +83,21 @@ def __init__(
         base: int = 10000,
         device: Optional[torch.device] = None,
     ):
-        """Rotary embeddings (Su et al., 2021) layer. The rotary embedding
-        will be precomputed for up to 'seq _len' positions. The embedding
+        """
+        Construct a rotary embedding module. The rotary embedding
+        will be precomputed for up to ``seq _len`` positions. The embedding
         will be recomputed when a longer sequence is found in the input.
 
         :param width:
-            Rotary embedding dimensionality, must be even.
+            Rotary embedding dimensionality.
+            Must be even.
         :param seq_len:
             Number of positons to initially precompute.
         :param base:
-            The base used for Θ_i, determines the cycle length of the
-            embeddings.
-        :param device: Device on which the module is to be initialized.
+            The base used for ``Θ_i``.
```

Comment:
Sphinx also has support for LaTeX math:

https://sphinx-rtd-trial.readthedocs.io/en/latest/ext/math.html

```suggestion
            The base used for :math:`\theta_i`.
```

Reply:
Fixed.